### PR TITLE
Updated paw Graph to match figma

### PR DESCRIPTION
**Pull request by HarlenPostill**
Screenshot 2024-09-30 at 9 57 50 pm

### CODE REVIEW

File: src/webview/App.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ const App: React.FC = () => {
     };
   }, []);
 
-  const isWidthTooSmall = width < 250;
+  const isWidthTooSmall = width < 304;
```
Comment (HarlenPostill) [inline src/webview/App.tsx:30]: This is what the Figma kinda is based off

File: src/webview/components/Atoms/Button/Button.tsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ type ButtonProps = {
   variant?: 'filled' | 'outlined';
 };
 
-const Button: React.FC<ButtonProps> = ({ text, onClick, variant = 'filled' }) => {
```
Comment (HarlenPostill) [inline src/webview/components/Atoms/Button/Button.tsx:10]: Recommend not using React.FC anymore I was looking into it and its fully redundant in modern React now 😭

File: src/webview/components/Atoms/Header.tsx [hunk 3]
```diff
@@ -4,7 +4,7 @@ type HeaderProps = {
   title: string;
 };
 
-const Header: React.FC<HeaderProps> = ({ title }) => {
```
Comment (HarlenPostill) [inline src/webview/components/Atoms/Header.tsx:7]: Recommend not using React.FC anymore I was looking into it and its fully redundant in modern React now 😭

File: src/webview/components/Atoms/LastSync/LastSync.tsx [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import './LastSync.css';
 
-const LastSync: React.FC = () => {
```
Comment (HarlenPostill) [inline src/webview/components/Atoms/LastSync/LastSync.tsx:4]: Recommend not using React.FC anymore I was looking into it and its fully redundant in modern React now 😭

File: src/webview/components/Atoms/ViolationSeverity/ViolationSeverity.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useDictionary } from '../../../hooks/useDictionary';
+import './ViolationSeverity.css';
+
+interface ViolationSeverityProps {
+  level: number;
+  num: number;
+}
+
+const ViolationSeverity = ({ level, num }: ViolationSeverityProps) => {
```
Comment (HarlenPostill) [inline src/webview/components/Atoms/ViolationSeverity/ViolationSeverity.tsx:10]: Bonus component so we can reuse if needed

File: src/webview/components/Molecules/Tabs/Tabs.tsx [hunk 6]
```diff
@@ -6,7 +6,7 @@ interface TabProps {
   children: React.ReactNode[];
 }
 
-const Tabs: React.FC<TabProps> = ({ headers, children }) => {
```
Comment (HarlenPostill) [inline src/webview/components/Molecules/Tabs/Tabs.tsx:9]: Recommend not using React.FC anymore I was looking into it and its fully redundant in modern React now 😭

File: src/webview/components/Molecules/WaterLevelPawPrint/WaterLevelPawPrint.tsx [hunk 7]
```diff
@@ -8,7 +8,7 @@ interface WaterLevelPawPrintProps {
   value3: number;
 }
 
-const WaterLevelPawPrint: React.FC<WaterLevelPawPrintProps> = ({ value1, value2, value3 }) => {
```
Comment (HarlenPostill) [inline src/webview/components/Molecules/WaterLevelPawPrint/WaterLevelPawPrint.tsx:11]: Recommend not using React.FC anymore I was looking into it and its fully redundant in modern React now 😭

File: src/webview/components/Organisms/ViolationsOverview/ViolationsOverview.tsx [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+import ViolationSeverity from '../../Atoms/ViolationSeverity/ViolationSeverity';
+import { useDictionary } from '../../../hooks/useDictionary';
+
+interface ViolationsOverview {
+  A: number;
+  AA: number;
+  AAA: number;
+}
+
+const ViolationsOverview = ({ A, AA, AAA }: ViolationsOverview) => {
```
Comment (HarlenPostill) [inline src/webview/components/Organisms/ViolationsOverview/ViolationsOverview.tsx:15]: First Organism 🥳

File: src/webview/App.tsx (hunk 1 above)
Comment (HarlenPostill) [reply src/webview/App.tsx:30]: Sorry this is actually just the exact PX that all of the words of the tab goes from multi-line to a single line, so I figured that would be a good place to start.

**Comment by angelovvv** [comment]
image
image

1. the tab bar "pink line" needs a fix because the line is not following the width when extended. 

image

2. The paw logo line is getting smaller when the water stays the same size. One solution could be to have a fix minimum size for the paw logo line as well to match the minimum water size. 

I like the idea of the blurring on the resize side bar and the water animation is cool. The tab bar is so smoothly done (I really like it).

**Review by angelovvv — APPROVED** [review]

**Pull request merged**